test/common.bash

```diff
@@ -17,6 +17,29 @@ use_config() {
 
 # wrappers around "kubectl config" command
 
+get_namespace() {
+  local cur_ctx
+
+  cur_ctx="$(get_context)" || exit_err "error getting current context"
+  ns="$(kubectl config view -o=jsonpath="{.contexts[?(@.name==\"${cur_ctx}\")].context.namespace}")" \
+     || exit_err "error getting current namespace"
+
+  if [[ -z "${ns}" ]]; then
+    echo "default"
+  else
+    echo "${ns}"
+  fi
+}
+
 get_context() {
-    kubectl config current-context
+  kubectl config view -o=jsonpath='{.current-context}'
+}
+
+exit_err() {
+  echo >&2 "${1}"
+  exit 1
```

Comment:
I'm not sure if `exit 1` is the way to fail from a bats test.

Can you just try making the last command `false` here, and see if it fails properly? I suspect if all `set -e`s are correct, this failure should propagate.

Reply:
noted but turns out this wasn't necessary at all